Codeinwp/tweet-old-post
# Add Webhooks integration

## Summary 

Added webhooks as an option to share posts. 

## Screenshots

#### Tab view

![image](https://github.com/Codeinwp/tweet-old-post/assets/17597852/badae61f-4cc2-4214-ab09-906c70d07695)

#### Popup

![image](https://github.com/Codeinwp/tweet-old-post/assets/17597852/0a8bbd5a-abf6-4300-8a69-56da1c69f574)

![image](https://github.com/Codeinwp/tweet-old-post/assets/17597852/42404976-a69b-4072-b410-2c23ad1e6cdc)

#### Instant sharing

![image](https://github.com/Codeinwp/tweet-old-post/assets/17597852/d0d481d7-c0f7-456b-9a38-e8a8df592dbe)

#### Demo


https://github.com/Codeinwp/tweet-old-post/assets/17597852/1def452d-ee95-4a42-a2c2-f0d9e5ad9b81



## Testing

> [!NOTE]
> You will need a webhooks service.

> [!WARNING]
> To test the functionality use https://github.com/Codeinwp/tweet-old-post-pro/pull/537

1. Add the webhook via Popup
2. Share on post.
3. Check if the data is received.
